src/styles/header.scss

```diff
@@ -0,0 +1,57 @@
+.logo {
+  display: flex;
+  height: 40px;
+  width: 40px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-inline-start: 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+  padding: 0;
+  margin: 0 50px;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: $header-height;
+  text-decoration: none;
+  color: $text-color-primary;
+  font-size: 12px;
+  text-transform: uppercase;
+  position: relative;
+}
+
+.is-active::after {
+  content: '';
+  display: inline-block;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 2px;
+  background-color: $color-primary;
+}
+
+.is-active,
+.nav__link:hover {
+  color: $color-primary;
```

Comment:
The variable `$color-primary` should be defined in your variables file or elsewhere in your project to ensure it is recognized here.